components/raftstore-v2/src/operation/command/write/mod.rs

```diff
@@ -313,6 +315,35 @@ impl<EK: KvEngine, R: ApplyResReporter> Apply<EK, R> {
             //     .unwrap_or_else(move |e| fail_f(e,
             // DeleteStrategy::DeleteBlobs));
         }
+
+        self.ensure_write_buffer();
+        // Delete range may not fill anything in memtable if the range is fit well with
```

Comment:
I think it's not because the range fits well with SST, but the memtable is empty or its range is not overlap with the delete range.

Reply:
Good catch. It's also possible.